<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditMilestoneDialog</class>
 <widget class="QDialog" name="EditMilestoneDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Milestone</string>
  </property>
  <widget class="QLabel" name="milestoneNamelabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Task name:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonOk">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>30</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCancel">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>60</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="startdateTimeEdit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>160</y>
     <width>130</width>
     <height>22</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListView" name="totalPeoplelistView">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>101</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="startDateTimelabel">
   <property name="geometry">
    <rect>
     <x>566</x>
     <y>130</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Date/Time:</string>
   </property>
  </widget>
  <widget class="QListView" name="taskPeoplelistView">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>140</y>
     <width>101</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="insertPeoplepushButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="teeamres.qrc">
     <normaloff>:/icons/icons/bullet_arrow_right.png</normaloff>:/icons/icons/bullet_arrow_right.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="removePeoplepushButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>230</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="teeamres.qrc">
     <normaloff>:/icons/icons/bullet_arrow_left.png</normaloff>:/icons/icons/bullet_arrow_left.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="taskGrouplabel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>30</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select task group:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="taskGroupcomboBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="selectPeoplelabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>All people:</string>
   </property>
  </widget>
  <widget class="QLabel" name="workingPeoplelabel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;People working at this task:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="baseGrouplabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Group:</string>
   </property>
  </widget>
  <widget class="QLabel" name="baseMilestonelabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Milestone:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="baseGroupcomboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="baseMilestonecomboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="teeamres.qrc"/>
 </resources>
 <connections/>
</ui>
